aten/src/ATen/native/native_functions.yaml

```diff
@@ -4529,10 +4529,6 @@
 
 - func: is_pinned(Tensor self, Device? device=None) -> bool
   variants: method
-  dispatch:
-    NestedTensorCUDA, CUDA: is_pinned_cuda
-    MPS: is_pinned_mps
-    CompositeExplicitAutograd: is_pinned_default
```

Comment:
Ho sorry for the oversight on my end.
The implementation here is not a valid CompositeImplicitAutograd implementation (the default key when nothing is specified as it calls into non-aten ops (the context).

Please update to
```

  dispatch:
    CompositeExplicitAutograd: is_pinned
```

Reply:
Yes, updated.